src/midi/midi_mt32.cpp

```diff
@@ -367,6 +367,8 @@ void MidiHandler_mt32::Close()
 	if (!is_open)
 		return;
 
+	HaltSequence();
```

Comment:
I don't think we need this sequence for a built-in mt32emu.

Reply:
In theory mt32emu and FluidSynth gracefully halt their in-progress sequence (so the last notes are played followed by silence) when we shut them down. 

But in practice, we stop the mixer channel first before the request to close them, so the music is abruptly stopped.  Which is fine.. and it works 100% of the time.  But I was hoping by adding this message before stopping the channel, we could get a more graceful close of the actual audio stream before the channel's cutoff. 

But it's perhaps just wishful thinking, and we can. Just terminate it hard like we currently do.

Reply:
I agree. We shouldn't try to fix something that's not really broken. Future maintainers might think there's some deeper meaning behind injecting this halting sequence.

*IF* abrupt closing of mixer is going to be a problem, then we should implement the adjustment in mixer (so it could be shared for pc-speaker, sblaster, and other channels). However, I don't think we have a problem at all - OS-level mixer (PulseAudio, PipeWire, etc) is probably dealing with this already.

For ALSA, win32, CoreMIDI, etc - we do need to do this because (some) MIDI synthesisers don't detect the MIDI channel going completely dark (hardware generally can't, external FluidSynth 1.1 couldn't, external Timidity was detecting it and stopping the audio 2-3s after no new MIDI instructions arrived).